teacher.rb

```diff
@@ -26,14 +28,14 @@ def teach_stuff
     response
   end
 
-  def salary=(new_salary)
-    puts "This better be good!"
-    @salary = new_salary
-  end
+  #def salary=(new_salary)
+   # puts "This better be good!"
+    #@salary = new_salary
+  #end
 
-  def receive_raise(raise)
-    @salary += raise
-  end
+  #def receive_raise(raise)
+   # @salary += raise
+  #end
 
   def set_performance_rating(rating)
```

Comment:
This method is very similar across the two instructor classes.  How could you DRY it up?